CONTRIBUTING.md

```diff
@@ -31,123 +31,112 @@ Note that we use the Google CLA because the OpenConfig project is
-
-1.  One or more peers in the community may review the pull request.
+## Code Style
 
-1.  A feature profile repository maintainer will be reponsible for a final
-    review and approval. Only a feature repository maintainer can merge a pull
-    request to the main branch.
+All code should follow
+[Go Code Review Comments](https://github.com/golang/go/wiki/CodeReviewComments)
+and [Effective Go](https://go.dev/doc/effective_go) for writing readable Go code
+with a consistent look and feel.
 
-The aim of this process is maintain the model quality and approach that
-OpenConfig working group has strived for since its inception in 2014. Thank you
-for your contributions!
+All code and documentation should follow
+[Google developer documentation style guide](https://developers.google.com/style/word-list)
+for the use of inclusive language.
 
-## Test development style guide
+It is recommended that tests follow
+[Testing on the Toilet](https://testing.googleblog.com/) for best practices on
+design patterns.
 
-### Directory Organization
+## Directory Organization
 
 The directory tree is organized as follows:
 
 *   `cloudbuild/` contains google cloud build scripts for running virtual
     routers in containers on [KNE](https://github.com/google/kne)
 *   `feature/` contains definition and tests of feature profiles.
 *   `feature/experimental` contains new features and tests which are not yet
-    proven to be stable
+    categorized or not confirmed to pass.
 *   `internal/` contains packages used by feature profile tests.
 *   `proto/` contains protobuf files for feature profiles.
 *   `tools/` contains code used for CI checks.
 *   `topologies/` contains the testbed topology definitions.
 *   `yang/` contains autogenerated go structures.
 
-### Test Suite Organization
+## Test Suite Organization
 
 Test suites should be placed in subdirectories formatted like
-`feature/<featurename>/[<sub-feature>/]<tests|ate_tests|otg_tests>/<testname>/main_test.go`,
-for example:
-
-*   `feature/interface/` contains feature profiles for interfaces.
-*   `feature/interface/README.md` - documents the interface feature profile.
-*   `feature/interface/ate_tests/` contains the interfaces test suite using ATE
-    traffic generation.
-*   `feature/interface/otg_tests/` contains the interfaces test suite using OTG
-    traffic generation.
-*   `feature/interface/tests/` contains the interfaces test suite without
-    traffic generation.
-*   `feature/bgp/` container for BGP feature profiles and test suites.
-*   `feature/bgp/addpath` contains feature profiles related to BGP.
-*   `feature/bgp/addpath/README.md` documents the BGP feature profile.
-*   `feature/bgp/addpath/ate_tests` contains test code related to BGP Add Path
-    features.
-*   `internal/deviations` contains code which overrides standard tests where
+`feature/<featurename>/[<sub-feature>/]<tests|ate_tests|otg_tests|kne_tests>/<test_name>/<test_name>.go`.
+For example:
```

Comment:
should we add example of where configuration generation code goes?  This has been a subject of confusion in the past.

Reply:
Since we did agree to split the config gen library from the device tests, I'll update this section as part of that work.